pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -209,7 +209,7 @@ public synchronized void channelInactive(ChannelHandlerContext ctx) throws Excep
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
         super.exceptionCaught(ctx, cause);
-        LOG.warn("[{}] Got exception {} : Message: {} State: {}", remoteAddress, cause.getClass().getSimpleName(),
+        LOG.warn("[{}] Got exception {} : Message: {} State: {} Cause: {}", remoteAddress, cause.getClass().getSimpleName(),
```

Comment:
```suggestion
        if (ClientCnx.isKnownException(cause)) {
            LOG.warn("[{}] Got exception {} : Message: {} State: {}",
                    remoteAddress, cause.getClass().getSimpleName(), cause.getMessage(), state, cause);
        } else {
            LOG.warn("[{}] Got exception {} : Message: {} State: {}",
                    remoteAddress, cause.getClass().getSimpleName(), cause.getMessage(), state);
        }
```

To stop the IDEA complaining, you can make this change.